### DISCUSSION TITLE
Automatically optimize Optimize Images

### DISCUSSION THREAD

**Discussion by benmccann** [evidence-dev/evidence#2354] [Ideas]
**Bug Description**
We've been searching around the web for cool sites built with SvelteKit and Evidence is a nice one. While checking it out, I noticed some opportunities to improve your score on PageSpeed insights. I hope you don't mind me sharing a couple suggestions

**Severity**
- [X] Low - this is unsolicited feedback, so feel free to ignore. Hope it's helpful, but also understand if it's not a priority!

**Expected Behavior**
You've got a file `chunks/vendor-1b48cd19.js`. That indicates to me that you're on a somewhat old version of SvelteKit. Newer versions of Vite have improved the code splitting and the site will likely load a lot less JS when you update to the latest

The other area for opportunity is optimizing images. See https://kit.svelte.dev/docs/images for that. If you run into any trouble, I'd be more than happy to help.

**Comment by mcrascal** [comment]
Wow @benmccann thank you for all of your contributions to svelte kit, and for taking the time to look at our project. 

We are planning our upgrade to the latest kit right now, and we will definitely incorporate these suggestions.

**Comment by mcrascal** [comment]
We have upgraded to the latest svelte Kit, going to keep this open for the image optimization suggestion.
